Friendzymes/README.md

```diff
@@ -0,0 +1,101 @@
+- CatA9__LEFT_PARENTHESISBsub_CamR_RIGHT_PARENTHESIS (selection_marker) in 
+- Eco31IA_methyltransferase (CDS) in 
+- Eco31IB_methyltransferase (CDS) in 
+- Eco31I_restriction_enzyme (CDS) in 
+- EcoOri_ColE1pMB1pBR32 (origin_of_replication) in 
+- Eco_AmpR (selection_marker) in 
+- Eco_KanR (selection_marker) in 
+- Eco_TetR (selection_marker) in 
+- Eco_ccdA (CDS) in 
+- Eco_p15a_origin (origin_of_replication) in 
+- Eco_pRO1600_ColE1_origin (origin_of_replication) in 
+- Eco_sacB (CDS) in 
+- Ecoli_KanR (selection_marker) in 
+- Ecoli_SpecR (selection_marker) in 
+- IDL3_B_BbsI_3_APOSTROPHE_Identity_Linker (assembly_component) in 
+- IDL3_B_mBsaI_3_APOSTROPHE_Identity_Linker (assembly_component) in 
+- IDL3_C_BbsI_3_APOSTROPHE_Identity_Linker (assembly_component) in 
+- IDL3_C_mBsaI_3_APOSTROPHE_Identity_Linker (assembly_component) in 
+- IDL5_A_BbsI_5_APOSTROPHE_Identity_Linker (assembly_component) in 
+- IDL5_A_mBsaI_5_APOSTROPHE_Identity_Linker (assembly_component) in 
+- IDL5_E_BbsI_5_APOSTROPHE_Identity_Linker (assembly_component) in 
+- IDL5_E_mBsaI_5_APOSTROPHE_Identity_Linker (assembly_component) in 
+- Lambda_T1_Transcriptional_Terminator (terminator) in 
+- Lox71_select_Lox66 (recombination_signal_sequence) in 
+- PSL3_B_to_A3_BbsI_3_APOSTROPHE_Part_Switch_Linker (assembly_component) in 
+- PSL3_B_to_A3_mBsaI_3_APOSTROPHE_Part_Switch_Linker (assembly_component) in 
+- PSL3_F9_to_E2_BbsI_3_APOSTROPHE_Part_Switch_Linker (assembly_component) in 
+- PSL3_F9_to_E2_mBsaI_3_APOSTROPHE_Part_Switch_Linker (assembly_component) in 
+- PSL3_F9_to_F_BbsI_3_APOSTROPHE_Part_Switch_Linker (assembly_component) in 
+- PSL3_F9_to_F_mBsaI_3_APOSTROPHE_Part_Switch_Linker (assembly_component) in 
+- PSL3_F_to_E2_BbsI_3_APOSTROPHE_Part_Switch_Linker (assembly_component) in 
+- PSL3_F_to_E2_mBsaI_3_APOSTROPHE_Part_Switch_Linker (assembly_component) in 
+- PSL5_A_to_A2_BbsI_5_APOSTROPHE_Part_Switch_Linker (assembly_component) in 
+- PSL5_A_to_A2_mBsaI_5_APOSTROPHE_Part_Switch_Linker (assembly_component) in 
+- PSL5_F8_to_A2_BbsI_5_APOSTROPHE_Part_Switch_Linker (assembly_component) in 
+- PSL5_F8_to_A2_mBsaI_Part_Switch_Linker (assembly_component) in 
+- PSL5_F8_to_A_BbsI_5_APOSTROPHE_Part_Switch_Linker (assembly_component) in 
+- PSL5_F8_to_A_mBsaI_Part_Switch_Linker (assembly_component) in 
+- Pfu_Sso7d_noCtag_Bsub (CDS) in 
+- Pfu_ssod7_yeast (CDS) in 
+- Pichia_VPS13_3_APOSTROPHE_Homology (homologous_region) in 
+- Pichia_VPS13_5_APOSTROPHE_Homology (homologous_region) in 
+- Pichia_VPS8_3_APOSTROPHE_Homology (homologous_region) in 
+- Pichia_VPS8_5_APOSTROPHE_Homology (homologous_region) in 
+- PurTag_R5 (polypeptide_motif) in 
+- Rec3_Lox66 (recombination_signal_sequence) in 
+- Rec3_LoxP (recombination_signal_sequence) in 
+- Rec5_Lox71 (recombination_signal_sequence) _<span style="color:red">not included in distribution</span>_
+- Rec5_LoxP (recombination_signal_sequence) _<span style="color:red">not included in distribution</span>_
```

Comment:
Are these two intentionally not included, or do you want to update the build plan?

Reply:
I believe this is simply an unnoticed error in porting parts to the spreadsheet.